.github/workflows/push-as-image-to-ghcr.yml

```diff
@@ -19,16 +19,40 @@ jobs:
           - coco-as-grpc
           - coco-as-restful
           - rvps
+        verifier:
+          - all-verifier
+          - se-verifier
         include:
```

Comment:
We should add the following:

```diff
+        exclude:
+          - instance: ubuntu-latest
+            verifier: se-verifier
+          - instance: s390x
+            verifier: all-verifier
```

Reply:
Oh, thanks for the TIPs! I forced-push the commit, please take a look if it is right